dig/dig.go

```diff
@@ -120,6 +127,13 @@ func (g *Graph) Resolve(obj interface{}) (err error) {
 	return nil
 }
 
+// MustResolve calls Resolve and panics if an error is encountered
+func (g *Graph) MustResolve(obj interface{}) {
```

Comment:
We are lucky, there is a tool just for that:
https://godoc.org/golang.org/x/tools/cmd/gorename